mmcls/models/backbones/swin_transformer.py

```diff
@@ -183,11 +183,11 @@ def __init__(self,
         else:
             self.downsample = None
 
-    def forward(self, x, in_shape):
+    def forward(self, x, in_shape, downsample=True):
         for block in self.blocks:
             x = block(x, in_shape)
 
-        if self.downsample:
+        if self.downsample is not None and downsample:
```

Comment:
param `downsample` is a little bit confusing, might use something like `downsample_inside` or `do_downsample` or `downsample_before_out`

Reply:
Renamed to `do_downsample`.